drivers/spi/spi_ll_stm32.c

```diff
@@ -96,14 +96,14 @@ static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
 #endif
```

Comment:
You commit message needs to be fixed as well:

Fixes #21935

We enforce this format, once the patch is merged the issue it then automatically closed (some script looks up for this format)

Reply:
> You commit message needs to be fixed as well:
> 
> Fixes #21935
> 
> We enforce this format, once the patch is merged the issue it then automatically closed (some script looks up for this format)

Done